vmdb/product/toolbars/container_group_center_tb.yaml

```diff
@@ -16,9 +16,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"container_group")}'
       :title: 'Edit this #{ui_lookup(:table=>"container_group")}'
       :url: '/edit'
+      :enabled: 'false'
```

Comment:
Because we will enable them eventually.  Right now I just wanted them to be grayed out.